code/modules/paperwork/photography.dm

```diff
@@ -356,3 +368,4 @@
 	pixel_x = P.fields["pixel_x"]
 	pixel_y = P.fields["pixel_y"]
 	photo_size = P.fields["size"]
+
```

Comment:
```diff
	photo_size = P.fields["size"]
- 
```
(It won't let me 'suggest' it properly, but this is to remove the extra empty line at the end of the file.)